apps/docs/src/app/core/component-docs/select/examples/select-adding-example/select-adding-example.component.html

```diff
@@ -3,7 +3,7 @@
     <button fd-button (click)="removeOption()">Remove Option</button>
 </div>
```

Comment:
Is this correct PR? Whey are you dealing with select? It just means that you havent properly rebased master adn you are sumbitting this as change.  this should not be here all related select

Reply:
I was rebasing from master at the exact time of your comment, not sure where this file came from, but i don't see it anymore. Please check